crypto-js/index.d.ts

```diff
@@ -3,9 +3,6 @@
 // Definitions by: Michael Zabka <https://github.com/misak113/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export = CryptoJS;
-export as namespace CryptoJS;
```

Comment:
I create `aes.d.ts`, and it works fine
```
import { AES } from './index'
export = AES;

```
But when I create `enc-base64.d.ts`, it throw `Cannot find namespace 'enc'`, how to fix it?
```
import { enc } from './index'
export = enc.Base64;
```